doc/releases.md

```diff
@@ -2,14 +2,71 @@
 
 ## Version 1.20
 
+### Version 1.20.1
+
+**February 19, 2025**
+
+This micro release includes a number of bug fixes, enhancements, and compatibility updates.
+Many thanks to the new contributors @spacetimeengineer, @flying-sheep, and @Azaya89, as well as the returning contributors @stanwest, @ahuang11, @maximlt, and @hoxbro
+
+Enhancements:
+
+- Change `param.ObjectSelector` to `param.Selector` ([#6444](https://github.com/holoviz/holoviews/pull/6444))
```

Comment:
This feels more like a compatibility change.

Reply:
Moved down to compatibility section.